### DISCUSSION TITLE
Conditional actions undocumented

### DISCUSSION THREAD

**Discussion by petrbrzek** [statelyai/xstate#3241] [Q&A] [answered]
Hi, I wanted to ask if [Conditional actions ](https://github.com/statelyai/xstate/pull/1076) that are merged are considered stable, or not. Also, there is nothing in the docs about it. Thanks

**Comment by davidkpiano** [comment, marked as answer]
This is documented as the [**Choose action**](https://xstate.js.org/docs/guides/actions.html#choose-action)

**Reply by petrbrzek** [reply]
Thanks, I couldn't find it. 🙂
